server/pcoippoolmanager/settings.py

```diff
@@ -78,12 +79,13 @@
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'postgres',
         'USER': 'postgres',
-        'HOST': 'db',
+        # TODO change host to proper one
```

Comment:
Who is supposed to do that?

Reply:
DevOps, because I don't know how credentials are supposed to look for the server we'll finally deploy on.